tests/osaka/eip7692_eof_v1/eip6206_jumpf/test_nonreturning_validation.py

```diff
@@ -208,3 +210,44 @@ def test_jumpf_in_nonreturning(eof_test: EOFTestFiller, first: bool, code_prefix
 def test_callf_to_nonreturning(eof_test: EOFTestFiller, container: Container):
     """Test EOF validation failing due to CALLF to non-returning section."""
     eof_test(container=container, expect_exception=EOFException.CALLF_TO_NON_RETURNING)
+
+
+VALID: List[Container] = [
+    Container(
+        name="returning_section_aborts_jumpf",
+        sections=[
+            Section.Code(code=Op.CALLF[1] + Op.STOP, max_stack_height=1),
+            Section.Code(
+                code=Op.PUSH0 * 2 + Op.RJUMPI[4] + Op.POP + Op.JUMPF[2] + Op.RETF, code_outputs=1
+            ),
+            Section.Code(
+                code=Op.PUSH0 * 2 + Op.RJUMPI[1] + Op.RETF + Op.INVALID,
+                code_inputs=0,
+                code_outputs=1,
+            ),
+        ],
+    ),
+]
+
+
+def container_name(c: Container):
+    """Return the name of the container for use in pytest ids."""
+    if hasattr(c, "name"):
+        return c.name
+    else:
+        return c.__class__.__name__
+
+
+@pytest.mark.parametrize(
+    "container",
+    [
+        *VALID,
```

Comment:
I'm not a big fun of the VALID / INVALID lists. Why not just place the container here?